notebooks/106-speech-recognition-quantize-wav2vec2/106-speech-recognition-quantize-wav2vec2.ipynb

```diff
@@ -0,0 +1,626 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "be9fc64c",
+   "metadata": {
+    "id": "f7i6dWUmhloy"
+   },
+   "outputs": [],
+   "source": [
+    "download_file(\"https://huggingface.co/facebook/wav2vec2-base-960h/resolve/main/pytorch_model.bin\", directory=Path(MODEL_DIR)/'pytorch', show_progress=True)\n",
+    "download_file(\"https://huggingface.co/facebook/wav2vec2-base-960h/resolve/main/config.json\", directory=Path(MODEL_DIR)/'pytorch', show_progress=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5dca2fa0",
+   "metadata": {
+    "id": "ehX7F6KB0uPu"
+   },
+   "source": [
+    "Import all dependencies to load the original PyTorch model and convert it to the ONNX representation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eb2f6d66",
+   "metadata": {
+    "id": "r5as0_Yg0uQX",
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "BATCH_SIZE = 1\n",
+    "MAX_SEQ_LENGTH = 30480\n",
+    "\n",
+    "def export_model_to_onnx(model, path):\n",
+    "    with torch.no_grad():\n",
+    "        default_input = torch.zeros([1, MAX_SEQ_LENGTH], dtype=torch.float)\n",
+    "        inputs = {\n",
+    "            \"inputs\": default_input\n",
+    "        }\n",
+    "        symbolic_names = {0: \"batch_size\", 1: \"sequence_len\"}\n",
+    "        torch.onnx.export(\n",
+    "            model,\n",
+    "            (inputs[\"inputs\"]),\n",
+    "            path,\n",
+    "            opset_version=11,\n",
+    "            #do_constant_folding=True,\n",
+    "            input_names=[\"inputs\"],\n",
+    "            output_names=[\"logits\"],\n",
+    "            dynamic_axes={\n",
+    "                \"inputs\": symbolic_names,\n",
+    "                \"logits\": symbolic_names,\n",
+    "            },\n",
+    "        )\n",
+    "        print(\"ONNX model saved to {}\".format(path))\n",
+    "\n",
+    "torch_model = Wav2Vec2ForCTC.from_pretrained(Path(MODEL_DIR)/'pytorch')\n",
+    "onnx_model_path = Path(MODEL_DIR)/\"wav2vec2_base.onnx\"\n",
+    "if not onnx_model_path.exists():\n",
+    "    export_model_to_onnx(torch_model, onnx_model_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6911365",
+   "metadata": {
+    "id": "sNWDAGGd0uRt"
+   },
+   "source": [
+    "## Convert the ONNX Model to OpenVINO IR"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20aeea80",
+   "metadata": {
+    "id": "-6P0c_960uR5",
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "ir_model_xml = onnx_model_path.with_suffix(\".xml\")\n",
+    "ir_model_bin = onnx_model_path.with_suffix(\".bin\")\n",
+    "\n",
+    "if not ir_model_xml.exists():\n",
+    "    !mo --input_model $onnx_model_path --output_dir $MODEL_DIR --model_name wav2vec2_base --input inputs --input_shape [1,-1] --data_type FP16"
```

Comment:
Do we need all these MO parameters, like "--input", "--input_shape", "--data_type"? Can we minimize them and keep only required, like path to model and output folder?